From: Ben Hutchings <ben@decadent.org.uk>
Date: Tue, 28 Aug 2018 18:47:31 +0100
Subject: locking/lockdep: Delete unnecesary #include
Forwarded: https://lore.kernel.org/lkml/20180828203315.GD18030@decadent.org.uk/T/#u

Commit c3bc8fd637a9 ("tracing: Centralize preemptirq tracepoints and u
nify their usage") added the inclusion of <trace/events/preemptirq.h>.
liblockdep doesn't have a stub version of that header so now fails to
build.

However, commit bff1b208a5d1 ("tracing: Partial revert of "tracing:
Centralize preemptirq tracepoints and unify their usage"") removed the
use of functions declared in that header.  So delete the #include.

Fixes: c3bc8fd637a9 ("tracing: Centralize preemptirq tracepoints ...")
Fixes: bff1b208a5d1 ("tracing: Partial revert of "tracing: Centralize ...")
Signed-off-by: Ben Hutchings <ben@decadent.org.uk>
---
 kernel/locking/lockdep.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/kernel/locking/lockdep.c b/kernel/locking/lockdep.c
index e406c5fdb41e..dd13f865ad40 100644
--- a/kernel/locking/lockdep.c
+++ b/kernel/locking/lockdep.c
@@ -55,7 +55,6 @@
 
 #include "lockdep_internals.h"
 
-#include <trace/events/preemptirq.h>
 #define CREATE_TRACE_POINTS
 #include <trace/events/lock.h>
 
